### ISSUE TITLE
Can't calculate with the variables

### ISSUE THREAD

**Issue by cm86** [mwittig/pimatic-fronius-solar#3]
Hello,
when i will send the variables of the fronius device to another part of pimatic.
fronius3.powerGenerate is undefined
and
fronius1.CurrentPower is NULL

i will calculate with the actual price and will send them to a graph-page (volkszaehler)
but it won't work. 
Because the variables are NULL ore undefined.
In Pimatic they have a value

mfg
Chris

**Comment by mwittig** [comment]
Please post your device configuration for fronius1 and fronius3. Thanks

**Comment by mwittig** [comment]
NB. Attribute names start with a lower case. For example:

```
WHEN currentPower of fronius-1 was not updated for 30 s  THEN ...
```

**Comment by cm86** [comment]
Ok but i will have the value of currentPower.

Mit freundlichen Grüßen
Chris

**Comment by mwittig** [comment]
Did you use the sample config from the README? If not, please post your device config. Which value do you get form fronius3.powerGenerate in pimatic?

**Comment by cm86** [comment]
Yes i used the sample. But here is my config.

`{
      "id": "fronius1",
      "class": "FroniusInverterRealtimeData",
      "name": "Fronius Inverter",
      "host": "192.168.5.89",
      "deviceId": 1,
      "threshold": 50
    },
    {
      "id": "fronius3",
      "class": "FroniusPowerFlowRealtimeData",
      "name": "Fronius Inverter 3",
      "host": "192.168.5.89",
      "attributes": [
        "mode",
        "powerGenerate",
        "energyDay",
        "energyYear",
        "energyTotal"
      ],
      "xAttributeOptions": []
    }`

How i said. in Pimatic GUI it has values. But when easily set a variable...
It is undefined or NULL...

Here Some Pics of my GUI
![bildschirmfoto 2016-08-31 um 22 03 59](https://cloud.githubusercontent.com/assets/14919688/18144660/fb31e4e2-6fc7-11e6-8dd6-8767b9dec72a.png)

Here are the device.variables

![bildschirmfoto 2016-08-31 um 22 04 22](https://cloud.githubusercontent.com/assets/14919688/18144661/fb358390-6fc7-11e6-821b-bc95e2df6e03.png)

Here is a variable which is set to the value of the fronius1.powerGenerate.
Just to check if its working.

![bildschirmfoto 2016-08-31 um 22 15 00](https://cloud.githubusercontent.com/assets/14919688/18144743/6edbb77e-6fc8-11e6-9bf1-d404e99f3600.png)

And here the result

![bildschirmfoto 2016-08-31 um 22 08 00](https://cloud.githubusercontent.com/assets/14919688/18144662/fb35e326-6fc7-11e6-88bc-169fe6812754.png)

**Comment by mwittig** [comment]
Thanks, for the clarification. I am now able to reproduce the problem and I am working on a fix.

**Comment by mwittig** [comment]
Fixed in pimatic-fronius-solar@0.0.15
Please re-open ticket if the issue remains

**Comment by cm86** [comment]
Hello,
thanks for the quick reply.
These device is working now...

Now the same issue is located at fronius1.currentPower and fronius1.currentAmperage.

mfg
Chris

**Comment by mwittig** [comment]
Strange, works for me.

EDIT: Ooops. Sorry. Variable never gets updated - fix in progress

![fronius-solar-issue3-1](https://cloud.githubusercontent.com/assets/6117742/18159624/e1320c22-7029-11e6-8533-12bcfa014a3c.png)

**Comment by mwittig** [comment]
Fixed in pimatic-fronius-solar@0.0.16
Please re-open ticket if the issue remains
